vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -113,6 +116,16 @@
     @Nullable
     private VimeoAccount mVimeoAccount;
 
+    public interface Caller<DataType_T> {
```

Comment:
Our style is itself a modified version of google's java style around type names (https://google.github.io/styleguide/javaguide.html#s5.2.8-type-variable-names). Note that their style also allows for use of a single letter, something that we haven't built into our style guide... yet